### ISSUE TITLE
Example render command in readme gives size mismatch error.

### ISSUE THREAD

**Issue by andybak** [apchenstu/TensoRF#11]
Note that I think this is different to #2 as I'm just trying to render the examples without training them. I might be misunderstanding though.

I've downloaded the dataset and pretrained checkpoints for Synthetic Nerf.

> python train.py --config configs/lego.txt --ckpt checkpoints/lego.th --render_only 1 --render_test 1

gives

> size mismatch for basis_mat.weight: copying a param with shape torch.Size([27, 288]) from checkpoint, the shape in current model is torch.Size([27, 864]).

**Comment by apchenstu** [comment]
the default configs correspond to the VM checkpoint, maybe you are using the CP checkpoints

**Comment by andybak** [comment]
Yep. I had to download this instead: https://onedrive.live.com/?authkey=%21AKpIQCzsxSTyFXA&id=C624178FAB774B7%21230&cid=0C624178FAB774B7

The readme is very unclear on what needs to be downloaded and what steps to follow. It might be worth spelling it out a bit more.

**Comment by rhmaaa** [comment]
Thanks. It is very useful.
